apps/schools/migrations/0023_auto_20170630_0642.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've put into a sql file here (https://github.com/klpdotorg/dubdubdub/blob/olp/sql/update_enums.sql) which you can just run with `psql -h localhost -U klp -d dubdubdub -f sql/update_enums.sql`.